assets/js/pad.js

```diff
@@ -0,0 +1,19 @@
+function Pad() {
+    
+    this.sketch;
```

Comment:
sketch is set with pad.load(sketch)
________________________________
From: mmou [notifications@github.com]
Sent: Wednesday, April 15, 2015 1:06 AM
To: mmou/tacit
Cc: Megan R O'Leary
Subject: Re: [tacit] Toolbar (#7)


In assets/js/pad.js<https://github.com/mmou/tacit/pull/7#discussion_r28393656>:

> @@ -0,0 +1,19 @@
> +function Pad() {
> +
> +    this.sketch;


how do you set this Pad's sketch? should be a parameter for Pad, or there should be a function that allows you to access it.

—
Reply to this email directly or view it on GitHub<https://github.com/mmou/tacit/pull/7/files#r28393656>.